README.rst

```diff
@@ -66,6 +66,16 @@ ecosystem of existing plugins for popular frameworks such as `pylint-django`_ or
 .. _`pylint-django`: https://github.com/PyCQA/pylint-django
 .. _`pylint-sonarjson`: https://github.com/omegacen/pylint-sonarjson
 
+.. note::
```

Comment:
```suggestion
.. Plugins::
```

Or something similarly specific.

Reply:
See:
https://pylint--7512.org.readthedocs.build/en/7512/

Reply:
Oh sorry I may have misunderstood. I thought a paragraph named `plugins` would be more on-point but whichever way you prefer.

Reply:
Ah ok I see the different rendering for note. I think the main thing is to have all plugin-related info in the same note or paragraph etc. just to keep them close together.